collector/container.go

```diff
@@ -0,0 +1,258 @@
+// returns containers metrics from HCS
```

Comment:
This becomes a package-level comment if viewed in godoc, which is confusing. If we want to have this comment at all (I'm not sure it adds much information that isn't readily apparent from skimming the code), merge it with the struct comment for `ContainerMetricsCollector`

Reply:
Removed it.